cmd/auto-pause/auto-pause-hook/config.go

```diff
@@ -36,8 +35,8 @@ var (
 	webhookConfigName = "env-inject.zyanshu.io"
 )
 
-// get a clientset with in-cluster config.
-func getClient() *kubernetes.Clientset {
+// Create a clientset with in-cluster config.
```

Comment:
If not used outside this package keep them private

Reply:
Fixed.